confgenerator/config.go

```diff
@@ -372,15 +358,7 @@ type MetricsReceiver interface {
 }
 
 type MetricsReceiverShared struct {
-	CollectionInterval string `yaml:"collection_interval" validate:"required,duration=10s"` // time.Duration format
-}
-
-func (m MetricsReceiverShared) CollectionIntervalString() string {
-	// TODO: Remove when https://github.com/goccy/go-yaml/pull/246 is merged
-	if m.CollectionInterval != "" {
-		return m.CollectionInterval
-	}
-	return "60s"
```

Comment:
Did we just remove the `60s` default? Does it matter?

Reply:
+1

Reply:
I put 60 seconds as a value any time it looked like we relied on the default result of `CollectionIntervalString`, did I miss any?
I will change to using a constant instead of putting 60 seconds everywhere.

Reply:
Yeah, a constant would be preferable, thanks!